src/pages/admin/countries/index.tsx

```
@@ -68,30 +66,26 @@ const CountryCard = ({ country }: { country: Country }) => {
 const CountryRequestCard = ({ request }: { request: CountryRequest }) => {
-  const queryClient = useQueryClient();
-  
-  const approveMutation = useMutation({
-    mutationFn: approveCountryRequest,
-    onSuccess: () => {
+  const approveMutation = useApproveCountryRequest();
+  const rejectMutation = useRejectCountryRequest();
+
+  const handleApprove = async () => {
+    try {
+      await approveMutation({ requestId: request.id as any });
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)